xforce.json

```diff
@@ -2,18 +2,19 @@
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
     "name": "IBM X-Force Exchange",
     "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
-    "publisher": "AvantGarde Partners",
+    "publisher": "Splunk",
     "package_name": "phantom_ibmxforceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
-    "copyright": "Copyright (c) AvantGarde Partners 2017",
+    "license": "Copyright (c) 2021-2022 Splunk Inc.",
+    "copyright": "Copyright (c) 2021-2022 Splunk Inc.",
     "main_module": "xforce_connector.py",
     "app_version": "1.1.1",
     "utctime_updated": "2022-06-08T12:27:25.000000Z",
     "product_vendor": "IBM",
     "product_name": "IBM X-Force Exchange",
     "product_version_regex": ".*",
     "min_phantom_version": "5.2.0",
+    "fips_compliant": false,
```

Comment:
One of the parameter accepts md5 hash. Therefore, this app should remain non-fips compliant.